### ISSUE TITLE
Windows taskbar calendar stays on top of ShareX

### ISSUE THREAD

**Issue by abcasada** [ShareX/ShareX#4626]
ShareX doesn't seem to play nicely with the Windows calendar in the taskbar. The calendar seems to sit on top of a ShareX "session" or whatever you call the visuals displayed by ShareX on top of the desktop when you're choosing a region to capture. For example, if I set a screenshot delay in ShareX, then click ShareX's icon to begin a capture, then open the taskbar calendar during the delay, once the "session" begins, if I move my cursor over the calendar, I cannot begin my selection from a point within the calendar. If I click inside the calendar, I am interacting with the calendar instead of with ShareX. Also, if I click inside the calendar, it seems that from this point on I can no longer cancel a capture by hitting ESC. The only way to get out is to go ahead and capture something.

**Comment by Jaex** [comment]
Are you using debug build of ShareX?

**Comment by abcasada** [comment]
I don't believe so. I'm on version 13.0.1, and do not recall downloading a debug build.

I'm also on Windows 10 Pro, version 1909, OS build 18363.592

**Comment by Scrxtchy** [comment]
I believe this is fixed in the 13.1 release. I remember it being an issue and I can no longer replicate it upon installing the update

**Comment by abcasada** [comment]
@Scrxtchy, thanks for the info. I have installed 13.1.0 just now, and I find that the issue still exists on this computer.

Also, I installed 13.0.1 on another computer (Windows 10 Pro 1909, OS build 18363.657), and was able to replicate the issue there. Then I installed 13.1.0 on that computer, and the issue persists.

Here's a video showing the issue, in case it's helpful: https://imgur.com/a/Ce4uGuY
